src/env.rs

```diff
@@ -53,8 +52,14 @@ pub fn sub_remaining_point(env: &Env, points: u64) -> anyhow::Result<()> {
     Ok(())
 }
 
-// if get_string throws an exception abort for some reason is being called
-pub fn abort(env: &Env, message: StringPtr, filename: StringPtr, line: i32, col: i32) {
+/// if get_string throws an exception abort for some reason is being called
```

Comment:
what's the link with get string ?

Reply:
the abort called by webassembly write an error in utf-8.... and I'm not sure to be able to say much :rofl:

Reply:
could we write a better doc comment for the `abort` function? I do not know myself what's this purpose ...

Reply:
#30 I get it